app/api/public.py

```diff
@@ -42,9 +48,20 @@ def states_daily_query():
 
 @api.route('/public/states/daily', methods=['GET'])
 def get_states_daily():
-    current_app.logger.info('Retrieving States Daily')
+    flask.current_app.logger.info('Retrieving States Daily')
     latest_daily_data = states_daily_query().all()
-    return jsonify([x.to_dict() for x in latest_daily_data])
+    return flask.jsonify([x.to_dict() for x in latest_daily_data])
+
+
+@api.route('/public/states/daily/<string:state>', methods=['GET'])
```

Comment:
What would you think about making this `/public/states/<string:state>/daily` to match the current public API (e.g. https://covidtracking.com/api/v1/states/ca/daily.json)?

Reply:
oh definitely gp - changing this now